### PR TITLE
🐛 Source Hubspot: fix `CAT` tests

### PR DESCRIPTION
**Pull request by bazarnov**
## What
Based on this CAT test: https://github.com/airbytehq/airbyte/actions/runs/4815554992
need to  ignore certain fields for `companies` stream to pass the CAT

## How
- ignore `properties/hs_all_*` while `basic_read` to satisfy `expected records` check

### CODE REVIEW

**Review by artem1205 — APPROVED** [review]

File: airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml [hunk 1]
```diff
@@ -43,6 +43,8 @@ acceptance_tests:
         companies:
         - name: properties/hs_time_*
           bypass_reason: Hubspot time depend on current time
+        - name: properties/hs_all_*
+          bypass_reason: This property is floating from sync to sync
```
Comment (erohmensing) [inline airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml:47]: This is ignoring all `hs_all_whatever` fields? I find it hard to believe that all of them float if the data's not changing. E.g. these ones are probably stable: `"hs_all_accessible_team_ids": null, "hs_all_owner_ids": "52550153", "hs_all_team_ids": null,` Are there specific ones that are causing issues?

Comment (erohmensing) [reply airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml:47]: I don't understand why they could change though 😅 there won't be new owners or teams, etc. But now I'm also confused, as there are still `hs_all` fields in the expeceted records, so maybe this isn't ignoring what I think it is?

Comment (bazarnov) [reply airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml:47]: Ok, I'll remove this from `acceptance-test-config.yaml`

Comment (erohmensing) [reply airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml]: Just for posterity's sake, looking at the diff, I see some completely new `hs_all` fields, which is annoying when hubspot changes stuff, but I think its more of "hubspot is giving us new fields" and not that the data within that field is changing:

image

I think it makes sense to update the expected records to contain the new fields in that case.

**Pull request merged**